index.js

```diff
@@ -98,21 +98,29 @@
 
       // Remove the window resize event.
       $(window).off('resize.updateFixedHeaderPosition');
+
+      // Clear any column searches
+      this.$(".dataTable tfoot input").off();
```

Comment:
@stonecalvin and @nikimt, you may want to look into torso version 0.2.1. It has an _attach and _detach callback methods that are only invoked when the view is attached/detached to/from the DOM. This can be more useful/safer then the plug/unplug methods that happen across any renders no matter if it's on the DOM.